people.json

```diff
@@ -3425,14 +3425,18 @@
       "company": "",
       "pronouns": "He/Him",
       "location": "Virginia Beach, United States",
-      "linkedin": "",
+      "linkedin": "https://www.linkedin.com/in/bretefisher",
       "twitter": "https://www.twitter.com/bretfisher",
+      "bluesky": "https://bsky.app/profile/bretfisher.com",
       "github": "https://github.com/bretfisher",
       "wechat": "",
-      "website": "",
-      "youtube": "",
-      "slack_id": "Bret Fisher",
-      "languages": [],
+      "website": "https://www.bretfisher.com",
+      "youtube": "https://www.youtube.com/@BretFisher",
+      "slack_id": "USB3GD04E",
+      "email": "bret!bretfisher.com",
```

Comment:
```suggestion
      "email": "bret@bretfisher.com",
```

Reply:
I'd recommend leaving this blank (to eliminate spam) or setting to a valid email so that the UI/UX works as expected.

Reply:
Roger that!